### DISCUSSION TITLE
Support for line and area sources? Scale source size?

### DISCUSSION THREAD

**Discussion by Spnail66** [uataq/stilt#130] [Users]
In the application of the model for a period of time, it is found that the STILT model is suitable for point source pollution simulation. On the model introduction page, I can see the STILT model is a variant of the HYSPLIT model. However, the HYSPLIT model supports point, line, and area sources, as well as setting the scale (size) of pollution sources. If the STILT model is to support three types of pollution source settings, please tell me how to set it? At the same time, I would like to know if the model can set and simulate pollution diffusion results under different source sizes and vertical height layers, and how? Thank you!

**Comment by benfasoli** [comment]
STILT is generally used for time-reversed applications, where receptors are represented as backward in time point sources. If you'd like to use line and area sources to represent emission sources, HYSPLIT may be a better tool for your use case.

If you find that the point-based receptor model is a good fit for your analysis, I'd recommend starting with the tutorials available in the [STILT documentation](https://uataq.github.io/stilt/).

**Comment by Spnail66** [comment]
Thank you! I will try to use the HYSPLIT model to implement the above functions.
